### PR TITLE
cleaned up and simplified rewards and other txs

### PR DESCRIPTION
**Pull request by jbibla**
**Description:**
- significantly reduced code complexity and lines of code
- used existing styles instead of Charlie's new styles for responsive designs
- changed styling of meta area to accommodate more data
- little hover effect is fun 🙃 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
@Bitcoinera i know that you were following parts of charlie's designs to solve this — we're in a difficult spot because the new designs are not designed in a way that allows them to be implemented easily in the current design. i tried to address the underlying issues (responsiveness, information density) by using existing styles from the other txs and with as little code as possible (-605 lines).

multiple validators, multiple amounts closed desktop:
Screen Shot 2020-03-27 at 10 33 19 AM

multiple validators, multiple amounts open desktop:
Screen Shot 2020-03-27 at 10 33 31 AM

multiple validators, multiple amounts closed mobile:
Screen Shot 2020-03-27 at 10 33 48 AM

multiple validators, multiple amounts open mobile:
Screen Shot 2020-03-27 at 10 33 38 AM

multiple validators, one amount open mobile:
Screen Shot 2020-03-27 at 10 41 21 AM

multiple validators, one amount open desktop:
Screen Shot 2020-03-27 at 10 41 14 AM

File: tests/unit/specs/components/transactions/message-view/ClaimRewardsTxDetails.spec.js [hunk 1]
```diff
@@ -103,43 +103,4 @@ describe(`ClaimRewardsTxDetails`, () => {
     )
     expect(getValidatorsResponse).toEqual([])
   })
-
-  it(`returns false if the claim was from one validator and just one denom`, () => {
```
Comment (faboweb) [inline tests/unit/specs/components/transactions/message-view/ClaimRewardsTxDetails.spec.js]: those tests are not applying anymore?

Comment (faboweb) [reply tests/unit/specs/components/transactions/message-view/ClaimRewardsTxDetails.spec.js]: there is no change in the logic of the multiClaimShow function

Comment (faboweb) [reply tests/unit/specs/components/transactions/message-view/ClaimRewardsTxDetails.spec.js]: I see it has no effect anymore. Should the computed be removed?

**Pull request merged**